render-utils.js

```diff
@@ -0,0 +1,32 @@
+export function renderGoblin(goblin) {
+    const goblinEl = document.createElement('div');
+    const nameEl = document.createElement('p');
+    const hpEl = document.createElement('p');
+    const imgEl = document.createElement('img');
+
+
+    //goblinEl.classList.add('.monster');
+    
+    //let goblinEmoji = '';
+
+    // if (goblin.hp >= 1) {
+    //     goblinEmoji = '👹';
+    // } else if (goblin.hp === 0) {
+    //     goblinEmoji = '💀';
+    // }
```

Comment:
not a huge deal, but taking out this old, commented out code will make your file cleaner